int32 timestep
int8 robot_id
---
SignalData[] data
